test/testUtils.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+const path = require("path");
+const execa = require("execa");
+const { sync: spawnSync } = execa;
+
+const WEBPACK_PATH = path.resolve(__dirname, "../bin/cli.js");
+
+/**
+ * Description
+ *
+ * @param {*} testCase The path to folder that contains the webpack.config.js
+ * @param {*} args Array of arguments to pass to webpack
+ * @returns {Object} The webpack output
+ */
+function run(testCase, args = []) {
+	// const cwd = path.resolve(testCase);
+	const cwd = path.resolve(testCase);
+
+	const outputPath = path.resolve(testCase, "bin");
+	const argsWithOutput = args.concat("--output-path", outputPath);
+
+	const result = spawnSync(WEBPACK_PATH, argsWithOutput, {
+		cwd,
+		reject: false
+	});
+
+	// TODO: STRIP OUT THE OUTPUT
+	result.stdout = removeTimeStrings(result.stdout);
+
+	return result;
+}
+
+function runWatch(testCase, args = []) {
+	const cwd = path.resolve(testCase);
+
+	const outputPath = path.resolve(testCase, "bin");
+	const argsWithOutput = args.concat("--output-path", outputPath);
+
+	return new Promise(resolve => {
+		// try {
```

Comment:
You can do `Promise.resolve().then(() => execa(...))....` to avoid doing `new Promise`. Not sure if it's better or not